symphony-bdk-http/symphony-bdk-http-webclient/src/main/java/com/symphony/bdk/http/webclient/ApiClientWebClient.java

```diff
@@ -146,7 +147,9 @@ public ApiClientWebClient(final WebClient webClient, String basePath, Map<String
       if (unwrap instanceof ApiException) {
         throw (ApiException) unwrap;
       } if(e.getCause() instanceof ConnectTimeoutException){
```

Comment:
formatting is off (missing a new line and spaces)

Reply:
fixing it